examples/cactus-example-cbdc-bridging-backend/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change Log
```

Comment:
@LordKubaya I recommend deleting this file - changelog.md files are generated by the release automation at the time when we issue releases (I'm guessing this just stuck around as a copy paste thing)

Reply:
yeah that was it